Reorganize Parallel notebooks, add density notebook

jadolfbr opened the pull request:
This PR moves notebooks from the 'deleted' directory into the main repo.  We can test these and fix them up - but moving them into a different directory is bad form.  

This organizes all parallel rosetta execution notebooks into a new chapter, 13, with a re-written introduction. 

This PR also adds the density notebook that somehow did not get added before. 

@kathyle9 - can you run the student-notebook script in this branch or open another one for this?  Also - for the <before> and <after> clickable jumps at the beginning and end of each file - is this done manually or is automatic with your script?

jadolfbr commented:
Looks like we still need Loop modeling, and Membrane proteins

kathyle9 commented:
I will make a new PR for running the make-student script. That will also take care of the links at the bottom--no need to manually update them. Thanks, Jared!

jadolfbr commented:
@klimaj - I have addressed your concerns, and added a long setup instruction for serialization build, including creating a condarc file for credentials.  I have also shortened Brian's second notebook, working on the first.

jadolfbr commented on notebooks/01.00-How-to-Get-Started.ipynb:
```diff
@@ -138,21 +138,36 @@
+    "**Step 9.1:**\n",
     "\n",
-    "`conda env create -f environment.yml`\n",
+    "**For Chapter 15, Running PyRosetta in Parallel, you will need to use a specific version of PyRosetta that is built for parallelization.**  This is the serialization build.  Besides manually building it from the Rosetta C++ source code, the general way to obtain this is through the use of a `conda` environment. \n",
```
Not every notebook that uses the distributed stuff needs serialization at the moment.  2.08 doesn't, a lot of the others don't as well.  I will see which ones need it and which ones don't.

jadolfbr replied:
A lot of the notebooks that originally needed that layer, I modified so that they don't anymore - so that the layer would pretty much only be in one chapter.   I'll go through each one to make sure though.

jadolfbr replied:
I added it to the ones that needed it and switched the Kernel.  Ideally, one would setup the conda env, and then activate before running all the notebooks, just like we did at Code School.  Without this, we are asking for trouble...
